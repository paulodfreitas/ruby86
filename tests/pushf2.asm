.module m0
.pseg
    irmovl $100, %esp
    pushf
    subl %eax, %eax
	halt
.end
