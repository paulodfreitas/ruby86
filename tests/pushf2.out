@ 0000
30
84
00
01
00
00
a0
00
61
00
10
@ 000b
